### PR TITLE
fix:toggle on/off on session-speakers page

### LINKED ISSUE #6214: If the field will not be empty then why there is on-off button
**Describe the bug**

The on-off switch should not be there if the field is mandatory

**Screenshots**

Screenshot (118)

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #6214

### CODE REVIEW

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 1]
```diff
@@ -1,24 +1,6 @@
...
-              </label>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-
-  {{#if this.data.event.isSessionsSpeakersEnabled}}
+  {{#if this.data.event}}
```
Comment (ranjitkshah) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs:3]: NO!

Comment (progmatic-99) [reply app/templates/components/forms/wizard/sessions-speakers-step.hbs:3]: Can you explain??

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 2]
```diff
@@ -1,24 +1,6 @@
...
-                {{t 'Sessions & Speakers'}}
-              </label>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-
-  {{#if this.data.event.isSessionsSpeakersEnabled}}
```
Comment (MeghalBisht) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs:21]: Removing this doesn't help as now this makes it compulsory to add a session. The switch is needed & should work & should not give any error when toggled "off"

Comment (progmatic-99) [reply app/templates/components/forms/wizard/sessions-speakers-step.hbs:21]: Should i enable all the forms (ticket, sponsors, sessions) by default?? Is that the ideal behaviour?

Comment (MeghalBisht) [reply app/templates/components/forms/wizard/sessions-speakers-step.hbs:21]: No

**Comment by mariobehling** [comment]
As I do not understand the issue description and the issue is closed I am closing this PR without an issue now.

**Pull request closed without merging**